egs2/slue-voxpopuli/slu1/conf/tuning/train_asr_vq_apc.yaml

```diff
@@ -0,0 +1,88 @@
+specaug: specaug
```

Comment:
It seems to me that the change of many configs are quite similar, do we want to keep all of them? (I'm ok with it if that's for public benchmark, but ideally, it would be better if we could summarize them into fewer configs, which is also easier for users)

Reply:
Yes, I considered it, but similar configs (like the ones using pre-trained ASR) have different sets of hyperparameters (like warmup steps, learning rate) selected based on validation performance. Hence, grouping similar configs would reduce reproducibility, so I think it is better to keep all of the configs.